modules/configuration/test/TestPlan.md

```diff
@@ -27,13 +33,13 @@ these fields:
    - try deleting a field with the 'X' button. Press save at the bottom of the page. 
    Refresh the page and check that the field was in fact deleted
   [Manual Test]
-6. Check that by setting 'Sandbox' in the config.xml to 1, the config tag names 
+7. Check that by setting 'Sandbox' in the config.xml to 1, the config tag names 
 appear in grey below their labels.
   [Automation Test]
-7. Check that the fields overridden in the config.xml appear greyed out in the config 
+8. Check that the fields overridden in the config.xml appear greyed out in the config 
 module and not editable.
   [Automation Test]
-8. Verify that Help section content and Developer's guide is complete, accurate and 
+9. Verify that Help section content and Developer's guide is complete, accurate and 
```

Comment:
Side note, this doesn't seem like it's possible to Automate this... (line 44)

Reply:
@johnsaigle u might be right